zeppelin-web/src/app/notebook/paragraph/paragraph-control.html

```diff
@@ -13,7 +13,24 @@
 -->
 
 <div id="{{paragraph.id}}_control" class="control" ng-show="!asIframe">
-
+  <span ng-show="paragraph.runtimeInfos.jobUrl.values.length == 1 && paragraph.runtimeInfos.jobUrl.group == 'spark'">
```

Comment:
Shell we remove condition `paragraph.runtimeInfos.jobUrl.group == 'spark'` ? so other interpreter can use this feature, too?